docs/compilation.rst

```diff
@@ -29,7 +29,7 @@ Building Python wheel and (optionally) Docker image
 Change directory (``cd``) into ``docker`` directory and run ``./build.sh``. If needed, set the following environment variables:
 
 * PYVER - Python version used to create a docker image with DALI installed inside. The default is ``3.6``.
-* CUDA_VERSION - CUDA toolkit version (10 for 10.0 or 11 for 11.0). The default is ``11``. If the version is prefixed with `.` then any value ``XX`` can be passed and the user needs to make sure that Dockerfile.cudaXX.deps is present in `docker/` directory.
+* CUDA_VERSION - CUDA toolkit version (10.0 or11.0). The default is ``11.0``. If the version is prefixed with `.` then any value ``XX.Y`` can be passed and the user needs to make sure that Dockerfile.cudaXXY.deps is present in `docker/` directory.
```

Comment:
```suggestion
* CUDA_VERSION - CUDA toolkit version (10.0 or 11.0). The default is ``11.0``. If the version is prefixed with `.` then any value ``XX.Y`` can be passed and the user needs to make sure that Dockerfile.cudaXXY.deps is present in `docker/` directory.
```

Reply:
I don't understand the "If the version is prefixed with `.` " part.

Reply:
How about now?